src/main/java/seedu/letsgethired/logic/parser/AddCommandParser.java

```diff
@@ -40,8 +36,9 @@ public AddCommand parse(String args) throws ParseException {
         Role role = ParserUtil.parseRole(argMultimap.getValue(PREFIX_ROLE).get());
         Cycle cycle = ParserUtil.parseCycle(argMultimap.getValue(PREFIX_CYCLE).get());
         Status status = ParserUtil.parseStatus(argMultimap.getValue(PREFIX_STATUS).get());
+        Note note = new Note("");
```

Comment:
Should this use `ParserUtil.parseNote` instead? Unless if you haven't implemented it yet?